models/labels/blue_chip_collection.sql

```diff
@@ -0,0 +1,45 @@
+with nft_trades as (
+  select *
+  from {{ ref('nft_trades') }}
+),
+
+volume_info as (
+  select
+    nft_contract_address,
+    sum(eth_amount) as volume
+  from nft_trades
+  group by nft_contract_address
+  order by volume desc
+  limit 30
```

Comment:
As far as I know, Nansen calculates its blue-chip index using "last 7-day transactions". Using all history data may miss the trending as well.